datalad/tests/test_dochelpers.py

```diff
@@ -86,6 +86,10 @@ def met1(self, bu, **kwargs):
         def met_nodoc(self, **kwargs):
             pass
 
+        @borrowkwargs(methodname=A.met1)
+        def met_anothermet(self, **kwargs):
+            pass
```

Comment:
you could add `   # pragma: no cover` comment inline here to signal that this code doesn't need to be covered by tests